test/trento/application/integration/checks/checks_test.exs

```diff
@@ -3,6 +3,7 @@ defmodule Trento.Integration.ChecksTest do
   use Trento.DataCase
 
   import Mox
+  import Mock
```

Comment:
we want to remove mock in a near future and just rely on mox, could you please add an issue about this test to keep track of it?